.github/workflows/ci-core.yml

```
@@ -88,7 +88,7 @@ jobs:
           allow-extra-out-format-args: true
           args: --out-format checkstyle:golangci-lint-report.xml
       - name: Print lint report artifact
-        if: always()
+        if: needs.init.outputs.on_trigger_lint == 'true'
         run: test -f golangci-lint-report.xml && cat golangci-lint-report.xml
```

If we need to keep this as `if: always()` what if we just make this command not fail of the file is not found?
```suggestion
        run: test -f golangci-lint-report.xml-blah && cat golangci-lint-report.xml || true
```

---

Ah yeah I thought something like this might be necessary :+1:

---

I couldn't see the exit code locally

---

I don't think this makes sense, this is a specific test and cat on a golangci file, the golangci is set to only run if go changes occur so it makes sense this would only run if golangci ran.

---

But we need to run when linting fails too, just like the `Store lint report artifact` step.

---

Can our if condition be based on file existence?

---

I am missing something here. What case is there that we want this check when the go linter doesn't run?